kube/src/config/mod.rs

```diff
@@ -102,21 +154,15 @@ impl Config {
         let token = incluster_config::load_token()
             .map_err(|e| Error::Kubeconfig(format!("Unable to load in cluster token: {}", e)))?;
 
-        let mut headers = HeaderMap::new();
-        headers.insert(
-            header::AUTHORIZATION,
-            header::HeaderValue::from_str(&format!("Bearer {}", token))
-                .map_err(|e| Error::Kubeconfig(format!("Invalid bearer token: {}", e)))?,
-        );
-
         Ok(Self {
             cluster_url,
             default_ns,
             root_cert: Some(root_cert),
-            headers,
+            headers: HeaderMap::new(),
```

Comment:
We no longer have any headers in here by default, but I wasn't sure if we needed to keep it around or not as it is a `pub` field on the Config struct

Reply:
Makes sense, conceivable that users might need to tweak other values in an emergency.